### ISSUE TITLE
Can You adapt it to newest automatic webui ? img2img is buggy on this one

### ISSUE THREAD

**Issue by 1blackbar** [TheLastBen/fast-stable-diffusion#6]
im getting error with img2img on this particuilar revision you used and it errors out often with img2img requiring total reset of the webui so all settings need to be dialed again and again, it happens when you import image to inpaint , go to extras and then go back to inpaint , your image is gone and you cant upload new one ... so i think newest repo would work better, this one is unlucky one

**Comment by TheLastBen** [comment]
Yes it's easy, just delete the "sd" folder in your GDrive and it will automatically download the latest release.

**Comment by G-force78** [comment]
> Yes it's easy, just delete the "sd" folder in your GDrive and it will automatically download the latest release.

Hi, I just did this and now its stuck on !mv /content/fast-stable-diffusion/precompiled/_C_flashattention.1 /content/fast-stable-diffusion/precompiled/_C_flashattention.7z.001

**Comment by TheLastBen** [comment]
> 

Make sure you have enough space in your Gdrive, delete the whole folder from trash also and restart

**Comment by G-force78** [comment]
> > 
> 
> Make sure you have enough space in your Gdrive, delete the whole folder from Gdrive and trash also and restart

Its an empty drive that started with 15gb it should be plenty?

**Comment by TheLastBen** [comment]
I just tested it, it runs without a problem, make sure you use the original colab from this repo

**Comment by G-force78** [comment]
> I just tested it, it runs without a problem, make sure you use the original colab from this repo

Ok will gove it another go, thanks for your work. 

Why are here so many unneccessary image files by the way?  https://imgur.com/a/yK66rKf

**Comment by TheLastBen** [comment]
They probably come with the AUTOMATIC1111 repo as some form of examples.

**Comment by G-force78** [comment]
> They probably come with the AUTOMATIC1111 repo as some form of examples.

Right ok

Now getting this error

NameError                                 Traceback (most recent call last)
[<ipython-input-9-a663d1bfab0b>](https://localhost:8080/#) in <module>
     16   gpu = 'P100'
     17 
---> 18 if not os.path.exists('/content/gdrive/MyDrive/sd/stable-diffusion-webui/xformers/_C.so') and (gpu=='T4'):
     19   get_ipython().run_line_magic('cd', '/content/')
     20   get_ipython().system('git clone https://github.com/TheLastBen/fast-stable-diffusion')

NameError: name 'gpu' is not defined

**Comment by TheLastBen** [comment]
Your Colab is tunning without a GPU, make sure you choose GPU in runtime type, then restart everything (by deleting the folder again)

**Comment by G-force78** [comment]
> Your Colab is tunning without a GPU, make sure you choose GPU in runtime type, then restart everything (by deleting the folder again)

OK, well its actually launched the webUI this time so theres progress but I'm not deleting all that and waiting all over again I thought this was supposed to be a quicker alternative to voldemorts. Probably my lack of knowledge but I'm done thanks anyway.

**Comment by TheLastBen** [comment]
The colab is fast and simple, and it is a quicker alternative to voldemorts, don't blame the colab.
